.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,44 @@
+name: Feature request
+description: Suggest an idea for AdGuard Filters
+labels: ["Feature Request"]
```

Comment:
What about to add `Infrastructure` label?

Reply:
Do you mean change the name from feature request to Infrastructure ?

Reply:
I thought maybe two labels would be better for this template. But for now, it seems like one is enough.
`labels: ["Feature Request", "Infrastructure"]`